nengo/builder/connection.py

```diff
@@ -169,7 +171,7 @@ def get_prepost_signal(is_pre):
     # Sample transform if given a distribution
     transform = (conn.transform.sample(conn.size_out, conn.size_mid, rng=rng)
                  if isinstance(conn.transform, Distribution) else
-                 conn.transform)
+                 np.array(conn.transform))
```

Comment:
So that's the side benefit. The main reason we want the copy is so that if we're doing something that doesn't have an ensemble as the pre, the transform simply becomes `weights`, and then we don't want our weights signal value to be a view on `conn.transform`.